modules/hitl/src/views/full/index.jsx

```diff
@@ -132,6 +167,14 @@ export default class HitlModule extends React.Component {
     }
 
     const currentSessionId = this.state.currentSession && this.state.currentSession.id
+    const customSearcProps = {
```

Comment:
small typo here `customSearchProps`

Reply:
typo error fixed and committed the changes